fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePrimaryKeySegmentIndex.java

```diff
@@ -313,31 +312,6 @@ public Collection<Accountable> getChildResources() {
         }
     }
 
-    /**
-     * Reconcile primary key index entries for a segment.
-     * Delete those for a segment that is about to be deleted or add for a segment that has been filled without going through the wrapper.
-     * The primary keys are gotten from the stored fields, so this must complete and not yet have been removed.
-     * @param reader a reader for the target segment
-     * @param add {@code true} to add entries; {@code false} to delete them
-     * @throws IOException if the stored fields gets an error
-     */
-    @SuppressWarnings("PMD.CloseResource")
-    public void addOrDeleteDocumentPrimaryKeys(@Nonnull SegmentReader reader, boolean add) throws IOException {
```

Comment:
This is approximately the code that would be needed if one wanted unoptimized storage and deferred segment index entry deletes, right? But we have no plans to support that combination.